scripts/hiseqx/xdemuxtiles.batch

```diff
@@ -10,6 +10,8 @@
 
 set -e -o pipefail
 
+useprod
+
```

Comment:
I dont think we need this since we do the check in line 27-30

Reply:
You're right. Forgot to delete this!